### ISSUE TITLE
Handle focus and tabindex properly when confirming/canceling selections

### ISSUE THREAD

**Issue by haxxmaxx** [qlik-oss/sn-table#146]
Currently, when canceling or confirming a selection using the keyboard, focus is lost. This would disorient the keyboard user, and sometimes make it hard to navigate back into the table.

For cancel, this should just preserve the focus. In theory, react should be able to just keep the focus when re-rendering. There will still be the same data in the table, so no need to calculate a new position.

For confirm, there are some options. Since confirming selections usually returns fewer rows than before, it is likely that the last focus cell does not exist anymore. Same thing if you make selections on page 2 or higher. Thus it make sense to reset the focus to the first cell or similar. Optimally not loosing focus, but at least setting all tabindexes correctly, so you can tab back if needed.

**Comment by haxxmaxx** [comment]
The fix was only for it to not lose focus on cancel. Still ended up in a weird place. The confirming part really needs to be fixed as well.
